.circleci/config.yml

```diff
@@ -262,25 +262,6 @@ jobs:
       - *build-Example
       - *save-cache
 
-  spm-integration-test:
-    parameters:
-      iOS:
-        type: string
-        default: "14.4"
-      xcode:
-        type: string
-        default: "12.4.0"
-    macos:
-      xcode: << parameters.xcode >>
-    environment:
-      HOMEBREW_NO_AUTO_UPDATE: 1
-    steps:
-      - checkout
-      - *prepare-mapbox-file
-      - *prepare-netrc-file
-      - *add-github-to-known-hosts
-      - run: scripts/build_spm_integration_test.sh << parameters.iOS >>
```

Comment:
> SPMTest project removed because breaks Carthage build.

I think you mentioned at one point that it breaks because Carthage somehow discovers and tries to build SPMTest.xcodeproj when building mapbox-navigation-ios as a dependency, even though the project is in a subsubfolder. Any idea why it affected SPMTest.xcodeproj but not PodInstall.xcodeproj or PodInstall.xcworkspace, which remains untouched?

Reply:
To be precise, I don't know all the details of Carthage's implementation. But let me try to make it a bit more clear what's happening because my previous explanation wasn't clear enough.

The error from Carthage is:
```
A shell task (/usr/bin/xcrun xcodebuild -project ./CarthageTest/Carthage/Checkouts/mapbox-navigation-ios/Tests/SPMTest/SPMTest.xcodeproj CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES -list) failed with exit code 74:
xcodebuild: error: Could not resolve package dependencies:
 Failed to clone repository ./CarthageTest/Carthage/Checkouts/mapbox-navigation-ios:
 fatal: repository './CarthageTest/Carthage/Checkouts/mapbox-navigation-ios' does not exist
```
even though `./CarthageTest/Carthage/Checkouts/mapbox-navigation-ios` path do exists. I've replaced full paths with "./" for brefity. 

I guess that the SPMTest project references the package in a hacky way (you can't do the same with the latest Xcode from the UI) and that causes the issues when Carthage tries to "open" this project. 

I found that digging deeper into the problem isn't worth it because I added "spm-test-job" jobs that run `swift test` commands. If this command succeeds, it almost guarantees that the SPM package is working. SPMTest project doesn't bring any more confidence. I would even say that it is misleading due to the way it uses the SPM package. Nobody will use our project in the way the SPMTest project uses it. 

So having said that above, I suggest just removing the SPMTest project and keep investing in `swift test`.